qa/suites/rgw/verify/tasks/s3tests.yaml

```diff
@@ -1,5 +1,6 @@
 tasks:
 - s3tests:
     client.0:
-      force-branch: ceph-master
+      force-branch: parquet_s3tests
+      git_remote: https://github.com/galsalomon66/
```

Comment:
please make sure this change doesn't merge to master

Reply:
:+1: